py3status/modules/xsel.py

```diff
@@ -24,25 +24,23 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 0.5
     command = 'xsel -o'
+    format = '{output}'
```

Comment:
maybe `{selection}` or some other name.  `{output}` feels too generic.

Also placeholder should be documented.